src/api/db/migrate/20161122121211_move_comments_data.rb

```diff
@@ -4,11 +4,21 @@ def change
       Comment.inheritance_column = nil
       Comment.find_each(batch_size: 5000) do |comment|
         /\AComment(?<type>.+)/ =~ comment.type
-        raise "Uncorrected type for comment with id #{comment.id}" unless ['Project', 'Package', 'BsRequest'].include? type
```

Comment:
@adrianschroeter didin't you want to delete the comments instead raising an exception when something is wrong with the comments?